### PR TITLE
create resources

### PR DESCRIPTION
**Pull request by ghoneycutt**

### CODE REVIEW

**Comment by ghoneycutt** [comment]
Any ideas why this is failing on Ruby 1.8.7? The error is not clear and I'm not able to reproduce locally.

File: Gemfile [hunk 1]
```diff
@@ -11,15 +11,18 @@ def location_for(place, fake_version = nil)
...
-  gem 'puppet_facts',            :require => false
-  gem 'json',                    :require => false
-  gem 'pry',                     :require => false
+  # rspec must be v2 for ruby 1.8.7
+  if RUBY_VERSION >= '1.8.7' and RUBY_VERSION < '1.9'
+    gem 'rspec', '~> 2.0'
+  end
+  gem 'rake',                              :require => false
+  gem 'rspec-puppet', '~> 1.0',            :require => false
+  gem 'puppetlabs_spec_helper', '>= 0.1.0' :require => false
```
Comment (tphoney) [inline Gemfile]: you are missing a comma.

**Comment by tphoney** [comment]
This also needs a rebase and could you squash your commits.

**Comment by ghoneycutt** [comment]
Passing ruby 1.8.7, though failing with future parser. This seems to be related to the testing setup and not this code. Could you confirm?

**Comment by bmjen** [comment]
@ghoneycutt Can you rebase this off master? I'll investigate the unit test failure.

**Comment by ghoneycutt** [comment]
@bmjen rebased

**Comment by bmjen** [comment]
@ghoneycutt I've submitted a PR to your branch that should fix the travis unit tests.

**Comment by HelenCampbell** [comment]
@ghoneycutt Any luck with this since @bmjen 's help?

**Comment by bmjen** [comment]
@ghoneycutt Thanks! Can you squash this all down? Then it will be ready for merge!

**Comment by ghoneycutt** [comment]
@bmjen squashed

**Comment by bmjen** [comment]
Thanks @ghoneycutt !

**Pull request merged**